src/index.js

```diff
@@ -3,6 +3,22 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import * as firebase from 'firebase';
+
+
+  // Your web app's Firebase configuration
+  var firebaseConfig = {
+    apiKey: "AIzaSyDaQdxs3hQ-nDVInjpdhgLbaleRIeIHn-Y",
```

Comment:
This should probably be protected through GitSecret in Settings
https://help.github.com/en/articles/virtual-environments-for-github-actions
This must be implemented by an Admin

Reply:
On it

Reply:
"Secrets are encrypted environment variables created in a repository **and can only be used by GitHub Actions.**"
I'm stuck on this issue: I'm not sure how to use them in the javascript file. The documentation suggests it's only usable during the GitHub Actions Workflow (and even there it's vague to me how to use them).

Reply:
After some study, either we:

1. Use something else than git secret
2. Just leave the api keys in there
3. Have a .env file (or properties file or w/e) that we give everyone seperate (AND NEVER PUSH)